gui/src/main/java/org/jboss/as/console/client/shared/runtime/naming/JndiViewManagement.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2011 Red Hat Inc. and/or its affiliates and other contributors
+ * as indicated by the @author tags. All rights reserved.
+ * See the copyright.txt in the distribution for a
+ * full listing of individual contributors.
+ *
+ * This copyrighted material is made available to anyone wishing to use,
+ * modify, copy, or redistribute it subject to the terms and conditions
+ * of the GNU Lesser General Public License, v. 2.1.
+ * This program is distributed in the hope that it will be useful, but WITHOUT A
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public License,
+ * v.2.1 along with this distribution; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+ * MA  02110-1301, USA.
+ */
+package org.jboss.as.console.client.shared.runtime.naming;
+
+/**
+ * @author Martin Stefanko
+ */
+public interface JndiViewManagement {
```

Comment:
There is no specific need for it. I made it to group possible further extensions or features to be added to the JndiPresenter class. In my opinion, it is no direct association to the funcionality the class should represent. There is very similar interface [TXMetricManagement](https://github.com/hal/core/blob/master/gui/src/main/java/org/jboss/as/console/client/shared/runtime/tx/TXMetricManagement.java).

Thank you for the review. If you wish, there is no problem to remove it.